files/enable-swap.sh

```diff
@@ -39,10 +41,10 @@ elif grep -E 'POWER9|POWER10' /proc/cpuinfo; then
         # and will stop working once we switch to p10. The setup should be done
         # generically, as stated in the comment above, so the large swap file
         # is created automatically upon the on_demand_powerful tag configuration.
-        truncate -s 310G "$file"
+        fallocate -l 310G "$file"
     else
         # regular builder
-        truncate -s 164G "$file"
+        fallocate -l 164G "$file"
```

Comment:
@FrostyX fallocate together with the file movement seems to help, can you re-review?